src/content/components/select/style.md

```diff
@@ -2,74 +2,99 @@
+
+---
+***
+> 
+![Open, disabled, and more information/help state examples for Select](images/select-style-3.png)
+
+_Examples of open, disabled, and help Select states_
+
+
+
+
+
 ## Typography
 
 Select text should be set in sentence case, with only the first word in a phrase and any proper nouns capitalized. Select text should be three words or less.
 
-| Property | Font-size       | Font-weight  | Type style |
-|----------|-----------------|--------------|---|
-| Label    | 14 / 0.875 | Semi-Bold / 600   | Zeta |
-| Label: inline select    | 14 / 0.875 |  Normal / 400   | - |
-| Text: input     | 14 / 0.875 | Normal / 400 | - |
-| Text: inline select     | 14 / 0.875 | Semi-Bold / 600 | Zeta |
-| Text: error message | 12 / 0.75 | Normal / 400 | - |
+| Class                                | Font-size (px/rem) | Font-weight     | Type style         |
+|--------------------------------------|--------------------|-----------------|--------------------|
+|`.bx--label`                          | 14 / 0.875         | Semi-Bold / 600 | `.bx--type-zeta`   |
+|`.bx--select--inline .bx--label`      | 14 / 0.875         | Normal / 400    | -                  |
+|`.bx--select-input`                   | 14 / 0.875         | Normal / 400    | -                  |
+|`.bx--select--inline.bx--select-input`| 14 / 0.875         | Semi-Bold / 600 | `.bx--type-zeta`   |
+|`.bx--form-requirement`               | 12 / 0.75          | Normal / 400    | `.bx--type-caption`|
 
 ## Structure
 
-| Property                    | px | rem   |
-|-----------------------------|----|-------|
-| Height                      | 40 | 2.5   |
-| Spacing: Label & field      | 8  | 0.5 |
-| Internal spacing            | 16 | 1     |
-| Groupings: vertical         | 32 | 2     |
-| Groupings: horizontal       | 16 | 1     |
-| Border bottom: enabled      | 1 | 0.0625rem |
-| Border bottom: focused      | 2 | 0.125 |
+### Select
+
+| Class                   | Property      | px / rem  | Spacing token |
+|-------------------------|---------------|-----------|---------------|
+|`.bx--select-input`      | height        | 40 / 2.5  | - |
+|`.bx--label`             | margin-bottom | 8 / 0.5   | $spacing-xs   |
+|`.bx--select-input`      | padding-right | 2.5 / 40  | $spacing-2xl  |
+|`.bx--select-input`      | padding-left  | 16 / 1    | $spacing-md   |
+|`.bx--select-input`      | box-shadow    | 1px       | - |
+|`.bx--select-input:focus`| box-shadow    | 2px       | - |
 
 ![Structure and spacing measurements for Select](images/select-style-2.png)
 _Structure and spacing measurements for Select | px / rem_
 
-### States
+### Recommended
 
-**Open:** Select menu styles are determined by browser. Each browser has a unique style. Shown in the example images is the Chrome select style.
+The following specs are not built into the Select component but are recommended by design as the proper amount of space to distance Select from another form element or component.
```

Comment:
"...the proper amount of space to distance Select from another form element or component." Something off with this sentence as well or am I misunderstanding? Its the "space to distance"  part that's throwing me off.